src/babel-plugin-transform-sx-tailwind/src/index.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+
+import template from '@babel/template';
+import * as t from '@babel/types';
+import murmurHash from '@adeira/murmur-hash';
+import { tailwindStyles } from '@adeira/sx-tailwind';
+
+module.exports = function sxTailwindBabelPlugin(): any {
+  const styles = [];
+  let tailwindCallee = '';
+  let sxtCallee = '';
+  let stylesVarName = 'styles';
+
+  return {
+    name: 'sx-tailwind-babel-transform',
+    visitor: {
+      ImportDeclaration(path) {
+        if (path.node.source.value === '@adeira/sx-tailwind') {
+          path.node.specifiers.forEach(({ imported, local }) => {
+            if (imported.name === 'tailwind') {
+              tailwindCallee = local.name;
+            } else if (imported.name === 'sxt') {
+              sxtCallee = local.name;
+            }
+          });
+
+          const sxImport = template.ast(`import * as sx from '@adeira/sx'`);
+          path.replaceWith(sxImport);
+        }
```

Comment:
Will it work with `import { create } from '@adeira/sx'` and `import { create as createStyles } from '@adeira/sx'`?

Reply:
Good point! Fixed and tested in `fixtures/sx-imported-alias/`